ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-3.72.yaml

```diff
@@ -0,0 +1,367 @@
+base_images:
+  eks:
+    name: automation-flavors
+    namespace: stackrox
+    tag: eks-0.3.1
+  golang-1.17.2:
```

Comment:
This needs to be `1.17.12` to match operator/Dockerfile on the target branch. (and in the 3 other places in this file). 
Marcin recently updated it, so you may have used a slightly stale openshift/release...master.yaml as the template.